Introduction.Rmd

```diff
@@ -137,64 +159,99 @@ Make sure you have a recent version of RStudio. You can check you have the right
 
 ```{r, eval = FALSE}
 install.packages("rstudioapi")
+# TODO: update this version number?
 rstudioapi::isAvailable("0.99.149")
 ```
 
-If not, you may need to install the preview version from <http://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
+If not, you may need to install the preview version from <https://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
```

Comment:
Can remove now

Reply:
I think the advice to use the preview version is still good, though, no matter what.